benchmarks/models.py

```diff
@@ -0,0 +1,77 @@
+import math
+
+import torch
+import torch.nn as nn
+
+
+class EmbeddingLayer(nn.Embedding):
+    def __init__(self, ntoken, ninp, initrange):
+        super().__init__(ntoken, ninp)
+        self.ninp = ninp
+        self.weight.data.uniform_(-initrange, initrange)
+
+    def forward(self, src):
+        return super().forward(src) * math.sqrt(self.ninp)
```

Comment:
minor: there is no need to sqrt ninp everytime in the forward since ninp is not changing.

Reply:
Done.